README.md

```diff
@@ -0,0 +1,2 @@
+<h1> BroncoDirectMe Backend </h1>
+<p> This repository contains all required code to drive the BroncoDirectMe aChrome Extencion. This includes Web scrapers for retrieving all professors from RateMyProfessor.com and APIs for displaying the informatin on the extension vlient.</p>
```

Comment:
looks good. We'll have more stuff later on to put. Please fix the spelling/spacing errors, thx :)